src/instructlab/model/list.py

```diff
@@ -77,21 +78,22 @@ def _analyze_dir(
     # walk entire dir.
     for root, _, files in os.walk(entry.as_posix()):
         normalized_path = os.path.normpath(root)
-        # Split the path into its components
-        parts = normalized_path.split(os.sep)
-        # Get the last two parts and join them back into a path
-        last_two_parts = os.path.join(parts[-2], parts[-1])
         # if this is a dir it could be:
         # top level repo dir `instructlab/`
         # top level model dir `instructlab/granite-7b-lab`
         # checkpoint top level dir `step-19`
         # any lower level dir: `instructlab/granite-7b-lab/.huggingface/download.....`
         # so, check if model is valid Safetensor, GGUF, or list it regardless w/ `--list-checkpoints`
         # if --list-checkpoints is specified, we will list all checkpoints in the checkpoints dir regardless of the validity
-        if is_model_safetensors(Path(normalized_path)) or is_model_gguf(
-            Path(normalized_path)
-        ):
-            actual_model_name = last_two_parts
+        for fname in [f for f in files if f.endswith("gguf")]:
```

Comment:
Are the above comments still relevant? Particularly the first three.

Reply:
this entire function has now been removed